sources/packages/backend/apps/db-migrations/src/sql/DisbursementSchedules/Add-tuition-remittance-effective-amount-col.sql

```diff
@@ -0,0 +1,7 @@
+-- Add tuition_remittance_effective_amount to disbursement_schedules table.
+ALTER TABLE
+    sims.disbursement_schedules
+ADD
```

Comment:
May be not a part of the PR or may be. The effective_amount and the awards(which are money values) are NUMERIC(8 , 2) but the tuition_remittance and tuition_remittance_effective_value are INT. Don't we eventually have one data type for the the amounts? @ann-aot @andrewsignori-aot @guru-aot @andrepestana-aot @sh16011993 

![image](https://user-images.githubusercontent.com/54600590/228660744-7d722871-3472-4e1e-b3bb-33e9f6d8ca1d.png)

Reply:
We had some back and forth about the money values actually having decimals or not. I would say that we can ensure that every money amount on DB has the ability to receive the decimals.
We also have different NUMERIC precisions (e.g. `NUMERIC(8, 2)` and `NUMERIC(7, 2)`).
I would say that we should normalize them all as `NUMERIC(8, 2)` which should allow 999,999.99.

Reply:
Yes, I can create a ticket to have this tracked.

Reply:
Ticket created https://app.zenhub.com/workspaces/student-information-management-system-5fce9df5aa1b45000e937014/issues/gh/bcgov/sims/1859